ansible/group_vars/production.yml

```
@@ -0,0 +1,36 @@
+  memory_warning_threshold: 80
+  memory_critical_threshold: 90
+  container_restart_threshold: 3
+  check_interval_minutes: 5
+
+# More conservative migration settings
+migration:
+  verify_data_integrity: true
+  create_backup_before_migration: true
+  migration_timeout_minutes: 120
+  require_manual_approval: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)